### PR TITLE
Add missing dependencies to VizGlow package

### PR DESCRIPTION
**Pull request by adamjstewart**

### CODE REVIEW

File: var/spack/repos/builtin/packages/vizglow/package.py [hunk 1]
```diff
@@ -39,20 +39,73 @@ class Vizglow(Package):
...
             url="file://{0}/VizGlow_v2.2alpha15-Linux-x86_64-R31October2016-Install".format(os.getcwd()))
 
+    # depends_on('mesa')  # TODO: mesa build doesn't work for me
+    depends_on('zlib')
+    depends_on('freetype')
+    depends_on('fontconfig')
+    depends_on('libxrender')
+    depends_on('xterm')
+    # Can't get mozjs to build, packagekit -> polkit -> mozjs
+    # depends_on('packagekit+gtk')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/vizglow/package.py]: I have WIP packages for packagekit, polkit, and mozjs. Wasn't sure whether or not they were worth adding.

File: var/spack/repos/builtin/packages/xterm/package.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
+class Xterm(AutotoolsPackage):
+    """The xterm program is a terminal emulator for the X Window System. It
+    provides DEC VT102 and Tektronix 4014 compatible terminals for programs
+    that can't use the window system directly."""
+
+    homepage = "http://invisible-island.net/xterm/"
+    url      = "ftp://invisible-island.net/xterm/xterm.tar.gz"
+
```
Comment (citibeth) [inline var/spack/repos/builtin/packages/xterm/package.py]: Is this another URL with an ever-changing hash, like TeXLive?  If so, is there an alternate location we can get a versioned tarball?  Maybe from Fedora?

Comment (adamjstewart) [inline var/spack/repos/builtin/packages/xterm/package.py]: Apparently I didn't look close enough at the ftp repo. There is a rolling release, but I think it's just a symlink to the versioned tarballs. Fixed now!

**Comment by adamjstewart** [comment]
Ping @tgamblin @becker33

**Comment by adamjstewart** [comment]
Ping ping @tgamblin

**Pull request merged**